src/res/translation/translation_it_IT.ts

```diff
@@ -583,7 +583,7 @@
     <message>
         <location filename="../../chatdlg.cpp" line="145"/>
         <source>Do you want to open the link &apos;%1&apos; in your browser?</source>
-        <translation>Vuoi aprire il link nel tuo Browser?</translation>
+        <translation>Vuoi aprire il link %1 nel tuo Browser?</translation>
```

Comment:
```suggestion
        <translation>Vuoi aprire il link &apos;%1&apos; nel tuo Browser?</translation>
```
?

Reply:
for me its ok! you need that i fix the file from my pull request? or is not necessary?

Reply:
You should be able to click the "Commit suggestion" button directly on Github to create a commit in your PR. :)